bundles/org.openhab.binding.haywardomnilogic/src/main/resources/OH-INF/thing/colorlogic.xml

```diff
@@ -39,30 +37,42 @@
 		</state>
 	</channel-type>
 
-	<channel-type id="currentShow">
+	<channel-type id="lightBrightness">
```

Comment:
This changed to `colorlogic` and related channels do not seem to be included in the upgrade isntructions. Is that on purpose?

Reply:
I added two new channels that are dynamically added if the user has the new colorLogic light.  I did not change the existing channels.